code/__defines/skills.dm

```diff
@@ -1,9 +1,9 @@
-#define SKILL_NONE     1
+#define SKILL_UNTRAINED     1
 #define SKILL_BASIC    2
-#define SKILL_ADEPT    3
-#define SKILL_EXPERT   4
-#define SKILL_PROF     5
-#define HAS_PERK       SKILL_NONE + 1
+#define SKILL_TRAINED    3
+#define SKILL_EXPERIENCED   4
+#define SKILL_MASTER     5
+#define HAS_PERK       SKILL_UNTRAINED + 1
```

Comment:
This has been in for a hot minute, but I haven't noticed any problems, nor heard anything reported. I think this is good to go. Could you make the indentation on these uniform (similar to the old defines) for consistency? After that, I think I'm good signing off on this.